plugins/updater/src/updater.rs

```diff
@@ -342,6 +365,18 @@ pub struct Updater {
 }
 
 impl Updater {
+    fn get_updater_installer(&self) -> Option<Installer> {
+        match bundle_type()? {
+            BundleType::Deb => Some(Installer::Deb),
+            BundleType::Rpm => Some(Installer::Rpm),
+            BundleType::AppImage => Some(Installer::AppImage),
+            BundleType::Msi => Some(Installer::Msi),
+            BundleType::Nsis => Some(Installer::Nsis),
+            BundleType::App => Some(Installer::App),
+            BundleType::Dmg => None,
```

Comment:
should this return Installer::App as well? (the updater does use the `.app` bundle for apps installed via .dmg)

Reply:
> On mac we also get BundleType::App.. I will change BundleTYpe::Dmg to default case as it better explains what happens.

I guess that didn't happen but maybe a comment would help prevent others from asking the same question i asked.